doc/ExtraDocuments/Fileformats.md

```diff
@@ -0,0 +1,248 @@
+
+For syntax: The name of each section is given by the file format (i.e. how the data is structured/arranged in the file). In paranthese one finds the type of field (2D or 3D) that is usually stored, as well as a common code that generates the files in question. Finally the file type (different from format) i.e. the `.extension` of the file (e.g. textfile, binary netcdf, ...) is listed as well.
+
+## gfiles/EQDSK format (axissymmetric 2D equillibrium, EFIT, textfile)
+
+[Original documentation from 1997](https://w3.pppl.gov/ntcc/TORAY/G_EQDSK.pdf) (note that there are some mistakes, e.g. `rcentr` does not exist) <br>
+
+Units: SI (NOTE: converters using gfiles as source may give out the quantities in cgs units after conversion) <br>
+entries per line: 5 (or less if end of points) <br>
+values format: ~ 16 characters (including sign and exponent notation) <br>
+
+While the general structure (see link above) should be the same for all files of that format, additional linebreaks and different formating of floating point numbers (number of digits) make it not possible to have one uniform reader routine for these textfiles. The user still has to choose the right reader routine for their present case, modify them accordingly or write their own.
+
+To give some orientation, for files which look like
+
+     Equil_2021_PMI_QH_mode_betap_1d04_li_1d02_Ip_18 3 360 400
+     18 20 8.9344 0 0
+     9.467 0.0583 302.6816 135.5856 -5.7463                1st block of values
+     -18270000.0013 -167.096 0 9.467 0       <- shot info           |
+     0.0583 -167.096 0 9.467 0                                      V
+     -5.09312766e+01 -5.09337203e+01 -5.09361562e+01 -5.09385842e+01 -5.09410043e+01 
+     -5.09434164e+01 -5.09458207e+01 -5.09482171e+01 -5.09506056e+01 -5.09529863e+01 
+    ...
+     -5.13398373e+01 -5.13398485e+01 -5.13398619e+01 -5.13398682e+01 -5.13398665e+01 
+                                                                                    <-\n
+     +1.92499100e+06 +1.91352411e+06 +1.90209288e+06 +1.89069842e+06 +1.87934049e+06 
+    ...
+
+where there is a line break in between the data blocks and the shot information (width in $R$ and $Z$ of box where data points are contained, leftmost $R$, ...) are listed in a different format than the quantity values in the subsequent blocks, the [eqdsk.py](https://github.com/itpplasma/libneo/blob/main/python/libneo/eqdsk.py) PYTHON class from libneo, can usually be used. You have to add libneo/python/ to your library [search path](https://note.nkmk.me/en/python-import-module-search-path/), see also [Libaris](https://gitlab.tugraz.at/plasma/info/-/tree/main/TOPIC_Compilation_and_Libraries/Libraries.md). Alternatively, you can just make a symbolic link in the same directory you want to import the reader.
```

Comment:
PYTHON is not FORTRAN 77, you can write it as Python
Also, should it be Libraries instead of Libaris?